kolena/dataset/dataset.py

```diff
@@ -71,7 +71,7 @@ class DatapointType(str, Enum):
     VIDEO = "DATAPOINT/VIDEO"
 
     @classmethod
-    def has_value(cls, item) -> bool:
+    def has_value(cls, item) -> bool:  # type: ignore
```

Comment:
Would it do to use `item: Union[str, "DatapointType"]`?

Reply:
Not quite but `Any` works and is probably the best signature here. Updated